pandas/tests/plotting/test_datetimelike.py

```diff
@@ -1104,6 +1104,33 @@ def test_time(self):
                     xp = time(h, m, s).strftime('%H:%M')
                 assert xp == rs
 
+    @pytest.mark.slow
+    @pytest.mark.xfail(reason="Unreliable test")
+    def test_time_change_xlim(self):
```

Comment:
Is there a reason you're just adding new code instead of separating out code from that test into a new one?

Reply:
Diff may be somewhat misleading but I essentially copied the entire test over and deleted the part from `set_xlim` onwards from original test

Reply:
Oh, that's weird...the diff was not right on that one.

Reply:
should we not see subtraction of code then as well?

Reply:
@jreback : I looked at it again, and I see why it's just an add.  `git` thinks @WillAyd just inserted a second copy of the first half of the original test.